README.org

```diff
@@ -228,7 +228,8 @@ Streaming is enabled by default. =(setq chatgpt-shell-streaming nil)= to disable
          (push `(,symbol
                  ,(car
                    (split-string
-                    (or (get (indirect-variable symbol)
+                    (or (documentation-property symbol 'variable-documentation)
```

Comment:
former one keep giving me the 

> ("~/.emacs.d/straight/build/chatgpt-shell/chatgpt-shell-openai.elc" . 252)

error when I was trying to generate the doc

Reply:
Thank you!